docs/tutorials/creating-your-first-substrate-chain/learn.md

```diff
@@ -0,0 +1,133 @@
+can do! Substrate provides you with a number of pre-built runtime modules
+collected in the **Substrate Runtime Module Library** (SRML).
+
+![Runtime Composition](assets/runtime.png)
+
+For example, the SRML includes a
+[Balances](https://substrate.dev/rustdocs/master/srml_balances/index.html)
+module that controls the underlying currency of your blockchain by managing the
+_balance_ of all the accounts in your system. 
+
+If you want to add smart contract functionality to your blockchain, you simply
+need to include the
+[Contracts](https://substrate.dev/rustdocs/master/srml_contracts/index.html)
+module.
+
+Even things like on-chain governance can be added to your blockchain by
+including modules like
+[Democracy](https://substrate.dev/rustdocs/master/srml_democracy/index.html),
+[Elections](https://substrate.dev/rustdocs/master/srml_elections/index.html),
+and
+[Collective](https://substrate.dev/rustdocs/master/srml_collective/index.html).
+
+The goal of this tutorial is to teach you how to create your own Substrate
+runtime module which will be included in your custom blockchain! The
+`substrate-node-template` comes with a template module that we will build your
+custom logic on top of.
+
+## Proof Of Existence Chain
+
+The custom logic we will add to your Substrate runtime is a Proof of Existence
+module. From [Wikipedia](https://en.wikipedia.org/wiki/Proof_of_Existence): 
+
+> Proof of Existence is an online service that verifies the existence of
+> computer files as of a specific time via timestamped transactions in the
+> bitcoin blockchain.
+
+Rather than uploading the entire file to the blockchain to "prove its
+existence", users submit a [hash of the
+file](https://en.wikipedia.org/wiki/File_verification), known as a file digest
+or checksum. These digests are powerful because huge files can be uniquely
+represented by a small hash value, which is efficient for storing on the
+blockchain. Any user with the original file can prove that this file matches the
+one on the blockchain by simply recomputing the hash of the file and compare it
+with the hash stored on chain.
+
+![File Hash](assets/file-hash.png)
+
+To add to this, blockchain systems also provide a robust account system. So when
+a file digest is stored on the blockchain, we can also which user uploaded that
+digest. This allows that user to later prove that they were the original person
+to claim the file.
+
+Our Proof of Existence module will expose two callable functions:
+
+* `create_claim` - Which allows a user to claim the existence of a file by
+  uploading a file digest.
+
+* `revoke_claim` - Which allows the current owner of a claim to revoke their
+  ownership.
+
+We will only need to store information about the proofs that have been claimed,
+and who made those claims.
+
+Sounds easy enough?
+
+If your node is finished compiling, then you are ready to move to the next
+section where we will interact with your Substrate node template!
```

Comment:
somethings it is spelled `Substrate node template` sometimes `Substrate Node template` sometimes `Substrate Node Template` - whichever we pick, we should stick with one (I personally think `Substrate node template` is the correct one)